examples/dts/sub-orchestrations-with-fan-out-fan-in/README.md

```diff
@@ -0,0 +1,102 @@
+# Portable SDK Sample for Sub Orchestrations and Fan-out / Fan-in
```

Comment:
Shouldn't this go directly under `examples`? Also, if we're updating the focus of the repo to just be DTS, then we don't need a `dts` subfolder under examples. All samples can assume DTS and have corresponding instructions for using the emulator.